eda/setup.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+###########################
+# This code block is a HACK (!), but is necessary to avoid code duplication. Do NOT alter these lines.
+import importlib.util
+import os
+
+from setuptools import setup
+
+filepath = os.path.abspath(os.path.dirname(__file__))
+filepath_import = os.path.join(filepath, '..', 'core', 'src', 'autogluon', 'core', '_setup_utils.py')
+spec = importlib.util.spec_from_file_location("ag_min_dependencies", filepath_import)
+ag = importlib.util.module_from_spec(spec)
+# Identical to `from autogluon.core import _setup_utils as ag`, but works without `autogluon.core` being installed.
+spec.loader.exec_module(ag)
+###########################
+
+version = ag.load_version_file()
+version = ag.update_version(version, use_file_if_exists=False, create_file=True)
+
+submodule = 'eda'
+install_requires = [
+    # version ranges added in ag.get_dependency_version_ranges()
+    'numpy',
+    'scipy',
+    'scikit-learn',
+    'pandas',
+    'matplotlib',
+    'missingno',
+    'phik',
+    'seaborn',
```

Comment:
I believe these packages are not covered by `get_dependency_version_ranges`. Can we ping them?

Reply:
We don't pin matplotlib in other modules. Added pins for other ones.